plugins/diff-highlight/prism-diff-highlight.css

```diff
@@ -0,0 +1,13 @@
+pre.diff-highlight > code .token.deleted:not(.prefix),
+pre > code.diff-highlight .token.deleted:not(.prefix) {
+	background-color: rgba(255, 0, 0, .1);
+	color: inherit;
+	display: block;
```

Comment:
Because of the `display: block`, it's compatible with the Funky theme. I already added a note for this on the plugin page but I would like to fix it. Any ideas?

<details>

![image](https://user-images.githubusercontent.com/20878432/61597325-5ca03180-ac0f-11e9-9d20-ed824415803a.png)

</details>

Reply:
Funky can do its "own" implementation of these styles, if they need to be different than the global ones.

Reply:
Done.
Not beautiful but ok, I guess

<details>

![image](https://user-images.githubusercontent.com/20878432/61597659-780d3b80-ac13-11e9-87f8-661312c90776.png)

</details>